### PR TITLE
Improvement/embedded hivemq jitpack

### PR DESCRIPTION
**Pull request by sauroter**
**Motivation**

Describes how to get embedded mode from jitpack.

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -144,14 +144,23 @@ HiveMQ Community Edition offers an embedded mode and a programmatic API for inte
...
+IMPORTANT: Until HiveMQ Community Edition 2020.3 is released embedded mode is only available from link:https://jitpack.io/#hivemq/hivemq-community-edition[jitpack.io].
+
 ==== Gradle
 
 If you use Gradle, include the following code in your `build.gradle` file.
 
 [source,groovy]
 ----
+	allprojects {
+		repositories {
```
Comment (SgtSilvio) [inline README.adoc:156]: please use 4 spaces instead of tabs

Comment (SgtSilvio) [reply README.adoc:156]: please also remove the root indentation

File: README.adoc [hunk 2]
```diff
@@ -144,14 +144,23 @@ HiveMQ Community Edition offers an embedded mode and a programmatic API for inte
...
 
+IMPORTANT: Until HiveMQ Community Edition 2020.3 is released embedded mode is only available from link:https://jitpack.io/#hivemq/hivemq-community-edition[jitpack.io].
+
 ==== Gradle
 
 If you use Gradle, include the following code in your `build.gradle` file.
 
 [source,groovy]
 ----
+	allprojects {
```
Comment (SgtSilvio) [inline README.adoc:155]: allprojects is not necessary, please remove

File: README.adoc [hunk 3]
```diff
@@ -144,14 +144,23 @@ HiveMQ Community Edition offers an embedded mode and a programmatic API for inte
...
+		repositories {
+			...
+			maven { url 'https://jitpack.io' }
+		}
+	}
+
 dependencies {
-    implementation group: 'com.hivemq', name: 'hivemq-community-edition-embedded', version: '2020.3'
+    implementation 'com.github.hivemq:hivemq-community-edition-embedded:master-SNAPSHOT'
```
Comment (SgtSilvio) [inline README.adoc:163]: imho this should be
```suggestion
    implementation 'com.github.hivemq.hivemq-community-edition:hivemq-community-edition-embedded:master-SNAPSHOT'
```

File: README.adoc [hunk 4]
```diff
@@ -163,12 +172,19 @@ If you use Maven, include the following code in your `pom.xml` file.
 ----
 <project>
     ...
+    <repositories>
+		<repository>
+		    <id>jitpack.io</id>
+		    <url>https://jitpack.io</url>
+		</repository>
+	</repositories>
```
Comment (SgtSilvio) [inline README.adoc:180]:
```suggestion
    <repositories>
        <repository>
            <id>jitpack.io</id>
            <url>https://jitpack.io</url>
        </repository>
    </repositories>
```

File: README.adoc (hunk 3 above)
Comment (sauroter) [reply README.adoc:163]: yes you are right

File: README.adoc (hunk 1 above)
Comment (sauroter) [reply README.adoc:156]: done

File: README.adoc (hunk 2 above)
Comment (sauroter) [reply README.adoc:155]: done

**Pull request merged**